vizro-core/examples/visual-vocabulary/README.md

```diff
@@ -81,7 +81,7 @@ The dashboard is still in development. Below is an overview of the chart types f
 | Parallel coordinates  | ✅     | Magnitude                | [Parallel coordinates plot with px](https://plotly.com/python/parallel-coordinates-plot/)  | [px.parallel_coordinates](https://plotly.com/python-api-reference/generated/plotly.express.parallel_coordinates.html)                                                                           |
 | Pictogram             | ❌     | Magnitude                |                                                                                            |                                                                                                                                                                                                 |
 | Pie                   | ✅     | Part-to-whole            | [Pie chart with px](https://plotly.com/python/pie-charts/)                                 | [px.pie](https://plotly.com/python-api-reference/generated/plotly.express.pie)                                                                                                                  |
-| Radar                 | ❌     | Magnitude                |                                                                                            |                                                                                                                                                                                                 |
+| Radar                 | ✅     | Magnitude                |                                                                                            |                                                                                                                                                                                                 |
```

Comment:
If you could add these links (see other charts), then that would be great!

```
[Radar chart with px](https://plotly.com/python/radar-chart/)
[px.line_polar](https://plotly.com/python-api-reference/generated/plotly.express.line_polar)
```

Reply:
Added links